packages/google_maps_flutter/example/test_driver/google_map_inspector.dart

```diff
@@ -4,8 +4,8 @@
 
 import 'package:flutter/services.dart';
 
-class GoogleMapTestController {
-  GoogleMapTestController(this._channel);
+class GoogleMapInspector {
```

Comment:
nit: add a doc comment explaining the role of this class.

Reply:
I'm confused about why this is its own class. Why not put methods on the `GoogleMapController`?